mifosng-android/src/main/res/values/strings.xml

```diff
@@ -639,7 +655,14 @@
         <item>groups</item>
         <item>loans</item>
         <item>savingsaccounts</item>
-    </string-array>
+    </string-array>Mechanical
```

Comment:
Why do we have this here ? Mechanical, ECE/EIE, Sciences ...?

I don't understand.